spdx/snippet.py

```diff
@@ -30,10 +30,10 @@ class Snippet(object):
      which this snippet is associated with. Mandatory, one. Type: str.
      - conc_lics: Contains the license the SPDX file creator has concluded as
      governing the snippet or alternative values if the governing license
-     cannot be determined. Mandatory one. Type: document.License or
+     cannot be determined. Mandatory one. Type: License or
```

Comment:
Same as in the other file: Let's try to be consistent and use `license.License` everywhere